scripts/linux/config-etc-hosts.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+
+SCRIPT_NAME="manage-hosts.sh"
+SCRIPT_PATH="/vagrant/scripts/linux"
+
+PLAYGROUND_NAME="$1"
+#PLAYGROUND_NAME="k8s-p9"
+
+MASTER_1="k8s-master-1"
+MINION_1="k8s-minion-1"
+MINION_2="k8s-minion-2"
+MINION_3="k8s-minion-3"
+
+MASTER_1_IP="192.168.0.10"
```

Comment:
This is not going to work if you use a different subnet. Additionally, this is not scalable if we add/remove hosts (TBH, we have this issue elsewhere as well)

Reply:
ok, see my comment at the end